projects/joda/TimeFuzzer.java

```diff
@@ -0,0 +1,28 @@
+/* Copyright (C) 2025 ISP RAS                                                    
```

Comment:
Change to Google copyright, that fuzztarget is from oss-fuzz

Reply:
done